### PR TITLE
feat: add Książę pasta

### PR DESCRIPTION
**Pull request by m4tx**

### CODE REVIEW

**Review by phoe — APPROVED** [review]
https://www.youtube.com/watch?v=MJqhb_2jSm8

**Comment by m4tx** [comment]
> https://www.youtube.com/watch?v=MJqhb_2jSm8

@phoe for context see the stowarzyszeniowy channel on Discord.

**Review by phoe — APPROVED** [review]

**Review by phoe — APPROVED** [review]

**Review by phoe — APPROVED** [review]

**Review by phoe — APPROVED** [review]

**Review by phoe — APPROVED** [review]

File: src/slash_commands/pasta/ksiaze.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+- Co ty tu robisz? - spytał czombiarza, którego zastał siedzącego w milczeniu przed stołem i baterią wałówki.
+- Dupam - odpowiedział ponuro czombiarz.
+- Co to znaczy "dupam"? - spytał Mały Książę.
+- To znaczy, że spędzam swe życie przed madżongiem, stresując się każdym wyborem i narzekając na Discordzie po przegranych - odpowiedział zamyślony czombiarz.
+- A dlaczego dupasz? - pytał dalej Mały Książe.
+- Aby zapomnieć - odpowiedział czombiarz.
+- O czym zapomnieć? - zaniepokoił się Mały Książę, który już zaczął mu współczuć.
+- Aby zapomnieć, że jestem społecznie upośledzony - stwierdził czombiarz, schylając głowę.
+- Dlaczego jesteś społecznie upośledzony? - dopytywał się Mały Książę, chcąc mu pomóc.
+- Jestem społecznie upośledzony, bo dupam - zakończył czombiarz rozmowę i pogrążył się w milczeniu.
```
Comment (Iipin) [inline src/slash_commands/pasta/ksiaze.txt:11]: we need to escape the dashes or else it will render as a Markdown list

**Review by phoe — CHANGES_REQUESTED** [review]
uuuu coverage spad

**Review by Iipin — CHANGES_REQUESTED** [review]

**Review by phoe — COMMENTED** [review]
kur

File: src/slash_commands/pasta/ksiaze.txt (hunk 1 above)
Comment (m4tx) [reply src/slash_commands/pasta/ksiaze.txt:11]: ugh

Comment (m4tx) [reply src/slash_commands/pasta/ksiaze.txt:11]: dorime https://github.com/riichi/chombot/pull/176

Comment (phoe) [reply src/slash_commands/pasta/ksiaze.txt:11]: interimo adapare dorime

**Pull request merged**